reader.go

```diff
@@ -88,19 +88,23 @@ func (e *ParseError) Unwrap() error { return e.Err }
 
 // These are the errors that can be returned in ParseError.Err.
 var (
+	// ErrTrailingComma extra delimiter at the end of line error
```

Comment:
I strongly recommend we do not modify this file because it is a 'clone' of the official reader. If we modify it, it will be much harder for us in the future to merge future official changes with our clone.

Can we somehow remove this file from the linter radar? I recall there is some flag (managed: true or alike), @george-e-shaw-iv can help here with a tag

Reply:
You would have to fake it being a generated file https://github.com/getoutreach/lintroller/blob/main/internal/common/common.go#L44

I don't really recommend doing this and if you do please do not use it outside of this specific use-case or advertise that this is a possibility.

Reply:
I looked up the history of this repo, folks have been modifying these files a lot already :(

Reply:
chatted with Wentao offline, we have already deviated from the source ... does not matter anymore. Hope one day we can switch to a diff csv reader or maybe golang fixes the original bug